src/App.tsx

```diff
@@ -5,12 +5,12 @@ import LoginPage from "./pages/LoginPage/LoginPage";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
 
-function App() {
-  const [activePage, setActivePage] = useState("login");
+function App(): React.ReactElement {
+  const [activePage, setActivePage] = useState<"login" | "home" | "menu">("login");
```

Comment:
Would be better to move it to enum or type alias and use it
Optional change for later